src/pages/chan/components/Comment/Comment.js

```diff
@@ -0,0 +1,57 @@
+import { React, useState } from 'react';
+import CommentList from '../CommentList/CommentList';
+import './Comment.scss';
+
+function Comment() {
+  const [userName] = useState('JinHee');
+  const [comment, setComment] = useState('');
+  const [feedCommets, setFeedComments] = useState([]);
+  const [isValid, setIsValid] = useState(false);
+  const post = e => {
+    e.preventDefault();
+    const copyFeedComments = [...feedCommets];
+    copyFeedComments.push(comment);
+    setFeedComments(copyFeedComments);
```

Comment:
how about
```suggestion
    setFeedComments([...feedCommets, comment]);
```

한 줄로 불변성도 유지하면서 setState를 해줄 수 있습니다!
자주 사용되는 패턴이니 알아두시면 좋을 것 같습니다!